app/actors/hyrax/actors/environment.rb

```diff
@@ -0,0 +1,28 @@
+module Hyrax
+  module Actors
+    class Environment
```

Comment:
I assume this is a new class introduced in the recent hyrax change? I'm just curious why we need a local instance? Did you customize this for our needs?